README.md

```diff
@@ -18,8 +18,30 @@ data pipeline.
 
 ## Getting Started
 
+### New customer?
+
+ - If you do not have a Microsoft alias, this SDK is not intended for your use. However, you might find the source code useful for understanding how Microsoft collects telemetry.
```

Comment:
It might be unclear what does "Microsoft alias" mean.

Reply:
Yeah I am not sure how to frame it better - intention is to use include employees and vendors/contractors.

Reply:
ok, shamelessly copied from - https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Exporter.OneCollector#onecollector-exporter-for-opentelemetry-net. Thanks @cijothomas. Let me know if this is fine.

Reply:
All credits to @CodeBlanch who made the original wording!